Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM node:latest
 MAINTAINER Alex Kern <alex@kern.io>
+
+ENV DISABLE_GA yes
```

Comment:
Instead of `DISABLE_GA`, how about we use an environment variable `GA_ACCESS_TOKEN` which is empty by default but if set, includes the GA snippet? That way the https://file.pizza GA token isn't hard-coded anywhere.